reference/constraints/Negative.rst

```diff
@@ -0,0 +1,79 @@
+Negative
+========
+
+Validates that a value is a negative number. To force that a value is a negative
+number or equal to zero, see :doc:`/reference/constraints/NegativeOrZero`.
+To force a value is positive, see :doc:`/reference/constraints/Positive`.
+
+==========  ===================================================================
+Applies to  :ref:`property or method <validation-property-target>`
+Options     - `groups`_
+            - `message`_
+            - `payload`_
+Class       :class:`Symfony\\Component\\Validator\\Constraints\\Negative`
+Validator   :class:`Symfony\\Component\\Validator\\Constraints\\LesserThanValidator`
+==========  ===================================================================
+
+Basic Usage
+-----------
+
+The following constraints ensure that:
```

Comment:
“The following constraint ensures that:”

It’s singular, isn’t it?

Reply:
It is. Fixed.